docs/install.md

```diff
@@ -9,33 +9,22 @@ Only **Linux** and **OSX** platforms are supported now. These instructions have
 * **Elixir** `=1.10.*` (check with `elixir --version`)
 
 ## Install prerequisite packages
+It will install common development tools, geth and postgres.
 
 ```
-sh bin/install
-```
-
-## Install PostgreSQL
-
-```
-sudo apt-get install postgresql postgresql-contrib
-sudo -u postgres createuser omisego_dev
-sudo -u postgres psql -c "alter user omisego_dev with encrypted password 'omisego_dev'"
-sudo -u postgres psql -c "alter user omisego_dev CREATEDB"
+sh bin/setup
 ```
 
 ## Install Erlang and Elixir
 
 Add the Erlang Solutions repo and install
 ```
-wget https://packages.erlang-solutions.com/erlang-solutions_2.0_all.deb
+wget https://packages.erlang-solutions.com/erlang-solutions_2.0_all.deb \
+&& sudo apt install ./erlang-solutions_2.0_all.deb \
+&& rm ./erlang-solutions_2.0_all.deb
+sudo apt-get update
 sudo apt-get install esl-erlang=1:22.3.1-1 elixir=1.10.2-1
-sudo apt-get install -y erlang-os-mon
-```
-
-## Install Geth
-Install Geth version 1.9.12 from the URL below.
-```
-https://geth.ethereum.org/downloads/
+sudo apt-get install -y erlang-os-mon erlang-parsetools
```

Comment:
```suggestion
sudo apt-get install -y erlang-os-mon erlang-parsetools erlang-tools
```

Reply:
cover - A coverage analysis tool for Erlang.
 cprof - A profiling tool that shows how many times each function
   is called. Uses a kind of local call trace breakpoints containing
   counters to achieve very low runtime performance degradation.
 eprof - A time profiling tool; measures how time is used in Erlang
   programs. Predecessor of fprof (see below).
 fprof - Another Erlang profiler; measures how time is used in Erlang
   programs. Uses trace to file to minimize runtime performance
   impact, and displays time for calling and called functions.
 instrument - Utility functions for obtaining and analyzing resource
   usage in an instrumented Erlang runtime system.
 make - A make utility for Erlang similar to UNIX make.
 tags - A tool for generating Emacs TAGS files from Erlang source
   files.
 xref - A cross reference tool. Can be used to check dependencies
   between functions, modules, applications and releases.